doc/base/classes.xml

```diff
@@ -38554,7 +38555,7 @@
 		Array of Colors
 	</brief_description>
 	<description>
-		Array of Color, can only contains colors. Optimized for memory usage, can't fragment the memory.
+		Array of Color, Contains colors. Optimized for memory usage, can't fragment the memory.
```

Comment:
To match the PoolIntArray below, replace the first comma with a period.

Also, below it's phrased as "[adjective] array", here it's described as "Array of [noun]"; should probably be the same for both, or at least pluralize Color to Colors.